### DISCUSSION TITLE
FOSS4G SOZip presentation slides

### DISCUSSION THREAD

**Discussion by aborruso** [sozip/sozip-spec#8] [General]
Hi @rouault ,
I have seen you FOSS4G [presentation](https://www.youtube.com/watch?v=8v42oc8qSMQ). You did a nice job with SOZip.
Thank you so much, you are always great.

I'm preparing a small blog post in Italian, to spread it.
I would like to have the slides: is there a public URL?

Thank you

**Comment by rouault** [comment]
yes, http://download.osgeo.org/gdal/presentations/SOZip%20%28FOSS4G%202023%29.pdf

**Reply by aborruso** [reply]
Thank you very much
